save_data_to_file.py

```diff
@@ -0,0 +1,55 @@
+import time
+import os
+import datetime
+
+import engine
+
+
+class DataSaver:
+    def __init__(self, num_of_games=10):
+        # # defines number of games and the size of a team (the size is 10 to the power of team_size) and it defines how
+        # many different matches will be played like: 2 - 10x10 and 100x100
+        self.num_of_games = num_of_games
+    #     self.team_size = team_size
+
+    @staticmethod
+    def create_path():
+        # # creates folder for results if it doesn't exist, if it exists then it is just setting it up
+        new_path = 'results'
+        if not os.path.exists(new_path):
+            os.makedirs(new_path)
+
+    def file_name(self):
+        # # defines saved file name
+        x = datetime.datetime.now()
+        game_name = f'{x.year}-{x.month}-{x.day}-{x.hour}{x.minute:03} - {self.num_of_games} games'
+        return game_name
+
+    # # sets the header of the files
+    # with open(f"results/{game_name}.csv", "a") as f:
+    #     f.write(f"Round number, Team 1 size,Team 2 size, Game rounds, Duration, Winning team")
+
+    def data_save(self, team1_size, team2_size):
```

Comment:
To jest bardzo złożona funkcja. 
Data_save to powinna być tylko ta część gdzie masz „with open”. Zbuduj sonie funkcje pomocnicze jeśli ich potrzebujesz.
A to jest run_and_save czy coś takiego

Reply:
ok